tests/foreman/maintain/test_maintenance_mode.py

```diff
@@ -92,6 +105,14 @@ def _finalize():
         data_yml = yaml.safe_load(f)
     assert len(enable_sync_ids) == len(data_yml[':default'][':sync_plans'][':disabled'])
 
+    if local_advisor_enabled:
+        assert 'All timers stopped' in result.stdout
```

Comment:
I was first confused why you assert something *before* calling any command (and assigning things to `result`), but then realized all "do something" calls use `result` -- I guess the whole test could use some better naming of the vars, but not necessarily in this PR.

Reply:
Updated the var nams as I don't think anyone will get to it later, might as well do it while I'm here:)